### ISSUE TITLE
Problems with IE

### ISSUE THREAD

**Issue by DouglasMurrel** [blueimp/jQuery-File-Upload#2390]
Hello,
I have this page:

<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>

<script src="/js/jquery.ui.widget.js"></script>

<script src="/js/jquery.iframe-transport.js"></script>

<script src="/js/jquery.fileupload.js"></script>

<a href="" onclick="$jq('#photo1').click();return false;"><img alt="photo" src="/images/dummie-photo.jpg" border=1></a>
<input type="file" name="photo1" id="photo1" style="visibility:hidden;position:absolute;">

<script>
var $jq=jQuery.noConflict();
$jq(function(){
  $jq('#photo1').fileupload({
    url: 'someurl.php',
    dataType:'html',
    replaceFileInput:false,
    send: function (e,data){
      $jq('#fullbar').css('display','block');
      $jq('#filled_bar').css('width','0');
    },
    progress: function (e, data) {
      var progress = parseInt(data.loaded / data.total * 100, 10);
      $jq('#filled_bar').css('width',progress + '%');
    },
    done: function (e,data){
      $jq('#fullbar').css('display','none');
    }
  });
});
</script>

<div id="fullbar" style="border:1px solid;width:200px;height:15px;margin:5px 0 5px 10px;display:none;">
  <div id="filled_bar" style="background-color:#00CC00;width:0%;margin:0;padding:0;"><img src='/img/empty.gif' width=1 height=15></div>
</div>

It works fine in Firefox, but in IE 'progress' function doesn't start. I have IE8 in IE7-compatible mode.
Where can be error?
